libs/domain/search/facet-value-navigation/facet-value-navigation.component.ts

```diff
@@ -35,9 +38,14 @@ export class SearchFacetValueNavigationComponent
       this.enableClear &&
       (featureVersion >= '1.2' ? this.dirty : this.selectedLength);
 
+    const syncKey = this.$options().syncCollapsibleState
+      ? this.heading?.replace(' ', '-').toLowerCase()
```

Comment:
Not the best option, heading can change according the locale, need something more stable, for example let component know about the facet's param